spec/repositories/marshallers/section_association_marshaller_spec.rb

```diff
@@ -64,16 +57,22 @@
     end
 
     it "decorates the entity with the attributes" do
+      allow(ManualWithSections).to receive(:new)
+      allow(SectionBuilder).to receive(:new).and_return(:section_builder)
+      allow(NullValidator).to receive(:new)
+      allow(ManualValidator).to receive(:new)
+
       marshaller.load(entity, record)
 
-      expect(decorator).to have_received(:call).
-        with(entity, sections: sections, removed_sections: removed_sections)
+      expect(ManualWithSections).to have_received(:new).with(:section_builder, entity, sections: [section], removed_sections: [removed_section])
+      expect(NullValidator).to have_received(:new)
+      expect(ManualValidator).to have_received(:new)
```

Comment:
Also I might be missing something, but I'm not sure the `NullValidator` does anything useful at all, does it?